### PR TITLE
Monthly -> Regular

### PR DESCRIPTION
**Pull request by svillafe**
## Why are you doing this?

Currently, all of out filenames which are related to monthly contributions are called monthly... Since we are able to do annual also a more sensible name would be regular. This PR changes all the `monthly` to `regular`.

## Todo in the future:
* Change StateMachine Name
* Update support workers model

### CODE REVIEW

File: assets/pages/regular-contributions/helpers/ajax.js [hunk 1]
```diff
@@ -50,7 +50,7 @@ function requestData(paymentFieldName: PaymentField,
   if (state.user.firstName !== null && state.user.firstName !== undefined
     && state.user.lastName !== null && state.user.lastName !== undefined
     && state.user.email !== null && state.user.email !== undefined) {
-    const monthlyContribFields: MonthlyContribFields = {
+    const regularContribFields: MonthlyContribFields = {
```
Comment (rupertbates) [inline assets/pages/regular-contributions/helpers/ajax.js]: Shouldn't this type change to RegularContribFields?

Comment (svillafe) [reply assets/pages/regular-contributions/helpers/ajax.js]: Yes! thanks for spotting that.

File: app/assets/AssetsResolver.scala [hunk 2]
```diff
@@ -25,4 +25,4 @@ class AssetsResolver(base: String, mapResource: String, env: Environment) {
 }
 
 case class AssetNotFoundException(assetPath: String)
-  extends Exception(s"Cannot find asset $assetPath. You should run `npm run build-dev`.")
\ No newline at end of file
+  extends Exception(s"Cannot find asset $assetPath. You should run `yarn run build-dev`.")
```
Comment (Ap0c) [inline app/assets/AssetsResolver.scala]: Don't need the `run` with yarn I think.

File: assets/pages/contributions-thankyou/contributionsThankyou.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-#monthly-contributions-thankyou-page, #oneoff-contributions-thankyou-page, #monthly-contributions-pending-page {
+#monthly-contributions-thankyou-page, #oneoff-contributions-thankyou-page, #regular-contributions-pending-page {
```
Comment (Ap0c) [inline assets/pages/contributions-thankyou/contributionsThankyou.scss]: `monthly-contributions-thankyou-page`

File: app/assets/AssetsResolver.scala (hunk 2 above)
Comment (svillafe) [reply app/assets/AssetsResolver.scala]: Yeap, it is optional.

**Comment by svillafe** [comment]
Selenium test passed locally ✅

**Comment by svillafe** [comment]
The first build failed due to an error in NVM.

**Pull request merged**